tests/integration_tests/security/test_spectre.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests ensuring SPECTREv2 mitigations are present."""
+
+import pytest
+
+from framework import utils
+from framework import defs
+
+
+@pytest.mark.timeout(100)
+def test_spectre_mitigations():
+    """
+    Run `spectre-mitigations` to check the kernel is compiled with SPECTREv2 mitigations.
+
+    @type: security
+    """
+    # Run command and raise exception if non-zero return code
+    utils.run_cmd("chmod a+x tests/integration_tests/security/spectre-mitigations", cwd=defs.FC_WORKSPACE_DIR)
```

Comment:
you sure this does what you want? It looks like you're just running `chmod a+x` on the `spectre-mitigations` script?

Reply:
If we go with the bash script, I' d go with @pb8o 's approach. Add the script in the repo with +x permissions and avoid doing `chmod` here.

Reply:
yeah I think it's clearer that way

Reply:
I've updated it to only use python.

Reply:
Ok, looks good to me now.